README.md

```diff
@@ -4,20 +4,25 @@
 
 [patir](http://patir.rubyforge.org) provides code to enable project automation tasks:
 
- * A logging format for ruby's built-in Logger
- * A command abstraction with a platform independent implementation for running shell commands and ruby code
- * Command sequences using the same command abstraction as single commands.
- * Configuration format for configuration files written in ruby.
+* an adjusted logging format for the built-in logger of Ruby
+* a command abstraction with a platform independent implementation for running
+  shell commands or Ruby code
+* sequences of commands using the same command abstraction as single commands.
+* a Configuration class and format for loading configuration files written in Ruby.
 
 ## Why?
-We've been using the same things again and again and can't be bothered to code it anew every time.
 
-The command abstraction has been used the most, the Logger defaults and formatting the least.
+Some same things are used again and again and shouldn't be rewritten every time.
```

Comment:
* Some of the same things are used ...

Reply:
I fixed the text as proposed by you :+1: